fm4tlp/tgbl_comment_subgraph_prep.py

```diff
@@ -91,6 +92,7 @@
 
 from utils import communities
 from utils import negative_sampler
+from tensorflow.compat.v1.io import *
```

Comment:
Please remove this import (we cannot ever accept module-wide imports i.e. with *). tf is already accessible via `import tensorflow.compat.v1 as tf` above. Call gfile with `tf.io.gfile` throughout. Thanks